middlewares/validators/userStatus.js

```diff
@@ -118,10 +123,7 @@ const validateMassUpdate = async (req, res, next) => {
 const validateGetQueryParams = async (req, res, next) => {
   const schema = Joi.object()
     .keys({
-      taskStatus: Joi.string()
-        .trim()
-        .valid(userState.IDLE)
-        .error(new Error(`Invalid state value passed for taskStatus.`)),
+      batch: Joi.boolean().valid(true).error(new Error(`Invalid boolean value passed for batch.`)),
```

Comment:
When passed as query params `batch` will be of type string. Did you test this if the type is being converted?

Reply:
Yes it gets auto converted. I tested with other strings it fails. for boolean it works as expected.